### PR TITLE
fix: use ref instead of querySelector for DOM manipulation

### PR DESCRIPTION
**Pull request by sravan-s**
QuerySelector is inaccurate when multiple channels are rendered parallel

This doesnt include root for dropdowns/fileviewer etc

### CODE REVIEW

File: src/smart-components/Channel/context/utils.js [hunk 1]
```diff
@@ -8,19 +8,19 @@ import { OutgoingMessageStates } from '../../../utils/exports/getOutgoingMessage
 const UNDEFINED = 'undefined';
 const { SUCCEEDED, FAILED, PENDING } = getSendingMessageStatus();
 
-export const scrollIntoLast = (intialTry = 0) => {
+export const scrollIntoLast = (intialTry = 0, scrollRef) => {
```
Comment (sravan-s) [inline src/smart-components/Channel/context/utils.js:11]: @HoonBaek  use this pattern in threading

Comment (HoonBaek) [reply src/smart-components/Channel/context/utils.js:11]: Looks good, thank you 🙏

Comment (HoonBaek) [reply src/smart-components/Channel/context/utils.js:11]: variable name should be `initialTry`?

**Review by HoonBaek — APPROVED** [review]
Looks good

File: src/smart-components/Channel/context/utils.js (hunk 1 above)
Comment (sravan-s) [reply src/smart-components/Channel/context/utils.js:11]: si... Will apply before merging

**Pull request merged**